modern/src/common/converter.js

```diff
@@ -1,13 +1,47 @@
-export const speedConverter = (value, unit) => {
+const speedConverter = (value, unit, back = false) => {
```

Comment:
I think these functions should just return `factor`. It will make the code shorted. No breaks needed.

Reply:
Done.